app/routes/events/components/EventAttendeeStatistics.js

```diff
@@ -0,0 +1,283 @@
+// @flow
+
+import { DistributionDataPoint } from 'app/components/Chart/utils';
+import type { Dateish, EventRegistration } from 'app/models';
+import DistributionPieChart from 'app/components/Chart/PieChart';
+import ChartLabel from 'app/components/Chart/ChartLabel';
+import { Flex } from 'app/components/Layout';
+import {
+  CartesianGrid,
+  Legend,
+  Line,
+  LineChart,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import moment from 'moment';
+
+interface RegistrationDateDataPoint {
+  name: string;
+  registrations: number;
+  unregistrations: number;
+}
+
+const ABAKOM_GROUP_IDS = [4, 5, 6, 7, 8, 9, 10, 11, 98, 231];
+const REVY_GROUP_IDS = [59, 119, 120, 121, 123, 124, 153, 154, 214, 222, 225];
```

Comment:
It is kinda ugly to hard code the group IDs here, since it can get outdated if we for instance add another Abakom/Revy group. if someone has any suggestions for how to improve this, please let me know. 👀

Reply:
Yeah, we do have `GroupTypeCommitee` and `GroupTypeRevue`, but group type isn't fetched for users in registrations so you'd have to make some backend changes, or do some extra API-calls.

Reply:
Yeah, I think the most sensible here if you want it, is to fetch `/api/v1/groups/?type=<komite/revy>` and use the data from that to find what group type people are in.

Reply:
I agree with Ludvig. If we add more committees I bet we will not remember to add them here

Reply:
Yeah, that sounds like a good solution. Moved it now ✅